### PR TITLE
Activating Open Collective

### PR DESCRIPTION
**Pull request by xdamman**
Please merge this pull request to activate your Open Collective https://opencollective.com/bower

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -91,10 +92,15 @@
     "test": "grunt test",
     "ci": "grunt travis",
     "coveralls": "coveralls",
-    "prepublish": "in-publish && echo 'You need to use \"grunt publish\" to publish bower' && false || not-in-publish"
+    "prepublish": "in-publish && echo 'You need to use \"grunt publish\" to publish bower' && false || not-in-publish",
+    "postinstall": "./node_modules/.bin/opencollective-postinstall || exit 0"
```
Comment (sheerun) [inline package.json]: Does `exit 0` work on windows?

**Comment by sheerun** [comment]
I cannot merge it, it breaks installs on node 0.10-0.12

**Comment by xdamman** [comment]
🤔 good question. Do you have a Windows machine to test this?

**Comment by sheerun** [comment]
@xdamman We use Appveyor. Ideally you you could test opencollective-postinstall for all node versions on osx/linux (travis) and windows (appveyor).

**Comment by sheerun** [comment]
@xdamman Do you plan to update this PR?

**Comment by xdamman** [comment]
Sorry no time to investigate Windows now so I just updated the package.json to remove the post install and replace it with a very simple "echo" in the console.

**Comment by thinkxl** [comment]
It may be worth to get it working as it should in Windows. I will try to get a Windows machine to test it out.

**Comment by sheerun** [comment]
Also, we can't use emoticons as some terminals break if one uses them: https://github.com/yarnpkg/yarn/issues/405

**Comment by xdamman** [comment]
@sheerun ah good to know. I've updated the `opencollective-postinstall` package to not show emoji on Windows: https://github.com/OpenCollective/opencollective-postinstall/commit/b068519b21e50bcc97119769d8dc2c6b825dc147

I've also tested it with PowerShell on Mac and it works fine. But if someone could test on an actual Windows machine that would be great! Thank you for helping improving this package, it will benefit all collectives! :-)

**Comment by xdamman** [comment]
So I made a bunch of changes to https://github.com/opencollective/opencollective-postinstall to make it work with Windows. It works but for some reason, I cannot make it work with node 0.10 that you are testing against. I'd need more time to investigate that (if anyone wants to chime in, help is welcome!)

In the meantime, I've just removed it and replaced it with a simple text without emoji.

File: package.json [hunk 2]
```diff
@@ -91,10 +91,16 @@
     "test": "grunt test",
     "ci": "grunt travis",
     "coveralls": "coveralls",
-    "prepublish": "in-publish && echo 'You need to use \"grunt publish\" to publish bower' && false || not-in-publish"
+    "prepublish": "in-publish && echo 'You need to use \"grunt publish\" to publish bower' && false || not-in-publish",
+    "postinstall": "npm run -s donate",
```
Comment (sheerun) [inline package.json]: bower can be installed with yarn, i'd rather not call it here

**Comment by xdamman** [comment]
Ah. How could we do this?
Isn't npm coming with node anyway? Meaning that even if you install Bower with yarn, wouldn't this command still work?

The reason for doing it this way is to avoid having the command showing once and then execute which ends up showing that message twice. That's not super elegant.

**Comment by sheerun** [comment]
something like `"postinstall": "node lib/postinstall.js"` would work everywhere

**Comment by xdamman** [comment]
I pushed a change to fix this but for some reason appveyor didn't trigger.
Can you check?

**Pull request merged**